packages/upload/stories/upload.stories.tsx

```diff
@@ -195,21 +194,3 @@ export const _RestrictFileName: StoryObj<UploadProps> = {
 //   color: 'success',
 // };
 // ProgressBar.storyName = 'progress bar';
-type FileListStoryProps = {
-  fileName1: string;
-  fileName2: string;
-};
-export const _FileListStory: StoryObj<FileListStoryProps> = {
```

Comment:
jw why we've split this component out into it's own doc but not the remaining components